### PR TITLE
Update film resistance used in Construction:FfactorGroundFloor effective thermal resistance of the fictitious insulation layer

### LINKED ISSUE #10116: Construction:FfactorGroundFloor horizontal downward thermal resistance is incorrect
Issue overview
--------------
This comment in DataHeatBalance::ComputeNominalUwithConvCoeffs:

    // PURPOSE OF THIS SUBROUTINE:
    // Calculate Nominal U-value with convection/film coefficients for reporting by
    // adding on prescribed R-values for interior and exterior convection coefficients
    // as found in ASHRAE 90.1-2004, Appendix A. Used in EIO and tabular reports.
    // ASHRAE 90.1-2004 Section A9.4.1 shows the following:
    //      R-value Condition
    //      All exterior conditions                        IP: 0.17  SI: 0.0299
    //      All semi-exterior surfaces                     IP: 0.46  SI: 0.0810
    //      Interior horizontal surfaces, heat flow up     IP: 0.61  SI: 0.1074 <-- 0.11 in SI
    //      Interior horizontal surfaces, heat flow down   IP: 0.92  SI: 0.1620 <-- 0.16 in SI
    //      Interior vertical surfaces                     IP: 0.68  SI: 0.1198

and 2009 ASHRAE Fundamentals handbook Ch 26 Table 1:

image

show that the Rfilm_in coefficient is incorrect (should be (0.11 + 0.16) / 2 = 0.135, and up/down are reversed in comment):

        // ASHRAE Handbook Fundamental 2005
        // Thermal resistance of the inside air film, m2.K/W. Average of 0.14 (heat flow up) and 0.11 (heat flow down)
        Real64 constexpr Rfilm_in(0.125); <-- should be 0.135
        // Thermal resistance of the outside air film used in calculating the Ffactor, m2.K/W. 0.17/5.678
        Real64 constexpr Rfilm_out(0.03);

Issue and suggested correction courtesy of Trane North America.

### Details
Some additional details for this issue (if relevant):
 - Platform (Operating system, version)
 - Version of EnergyPlus (if using an intermediate build, include SHA)
 - Unmethours link or helpdesk ticket number

### Checklist
Add to this list or remove from it as applicable.  This is a simple templated set of guidelines.
 - [ ] Defect file added (list location of defect file here)
 - [ ] Ticket added to Pivotal for defect (development team task)
 - [ ] Pull request created (the pull request will have additional tasks related to reviewing changes that fix this defect)

### PR DESCRIPTION
**Pull request by rraustad**
Pull request overview
---------------------
 - Fixes #10116
 - Update slab film resistance used for Construction:FfactorGroundFloor effective thermal resistance of the fictitious insulation layer.

**NOTE: ENHANCEMENTS MUST FOLLOW A SUBMISSION PROCESS INCLUDING A FEATURE PROPOSAL AND DESIGN DOCUMENT PRIOR TO SUBMITTING CODE**

### Pull Request Author
Add to this list or remove from it as applicable.  This is a simple templated set of guidelines.
 - [x] Title of PR should be user-synopsis style (clearly understandable in a standalone changelog context)
 - [x] Label the PR with at least one of: Defect, Refactoring, NewFeature, Performance, and/or DoNoPublish
 - [ ] Pull requests that impact EnergyPlus code must also include unit tests to cover enhancement or defect repair
 - [x] Author should provide a "walkthrough" of relevant code changes using a GitHub code review comment process
 - [x] If any diffs are expected, author must demonstrate they are justified using plots and descriptions
 - [ ] If changes fix a defect, the fix should be demonstrated in plots and descriptions
 - [ ] If any defect files are updated to a more recent version, upload new versions here or on DevSupport
 - [ ] If IDD requires transition, transition source, rules, ExpandObjects, and IDFs must be updated, and add IDDChange label
 - [ ] If structural output changes, add to output rules file and add OutputChange label
 - [ ] If adding/removing any LaTeX docs or figures, update that document's CMakeLists file dependencies

### ToDo:
 - [x] Update Eng. Ref.

### Reviewer
This will not be exhaustively relevant to every PR.
 - [ ] Perform a Code Review on GitHub
 - [ ] If branch is behind develop, merge develop and build locally to check for side effects of the merge
 - [ ] If defect, verify by running develop branch and reproducing defect, then running PR and reproducing fix
 - [ ] If feature, test running new feature, try creative ways to break it
 - [ ] CI status: all green or justified
 - [ ] Check that performance is not impacted (CI Linux results include performance check)
 - [ ] Run Unit Test(s) locally
 - [ ] Check any new function arguments for performance impacts
 - [ ] Verify IDF naming conventions and styles, memos and notes and defaults
 - [ ] If new idf included, locally check the err file and other outputs

### CODE REVIEW

File: src/EnergyPlus/HeatBalanceManager.cc [hunk 1]
```diff
@@ -4707,8 +4707,8 @@ namespace HeatBalanceManager {
         // This subroutine only gets called once in the GetConstructionData subroutine
 
         // ASHRAE Handbook Fundamental 2005
-        // Thermal resistance of the inside air film, m2.K/W. Average of 0.14 (heat flow up) and 0.11 (heat flow down)
-        Real64 constexpr Rfilm_in(0.125);
+        // Thermal resistance of the inside air film, m2.K/W. Average of 0.16 (heat flow down) and 0.11 (heat flow up)
+        Real64 constexpr Rfilm_in(0.135);
```
Comment (rraustad) [inline src/EnergyPlus/HeatBalanceManager.cc:4711]: A simple change to update film resistance to match ASHRAE Fundamentals data and also used elsewhere in E+ code.

Comment (rraustad) [reply src/EnergyPlus/HeatBalanceManager.cc:4711]: The only question I would have is if I should have used more precision with (0.1620 + 0.1074)/2 = 0.1347 (see issue description). Diffs I've seen are very small so maybe using existing precision is sufficient.

**Comment by rraustad** [comment]
And of course I should have checked the docs:

image

**Comment by rraustad** [comment]
Diffs are very small. Here's an eio example diff from ASHRAE901_ApartmentMidRise_STD2019_Denver (which uses Construction:FfactorGroundFloor):

    - Zone Sizing Information, G SW APARTMENT, Cooling, 1612.85150, 1935.42180, 0.14448, 0.17337, DENVER-AURORA-BUCKLEY.AFB_CO_USA ANN CLG .4% CONDNS DB=>MWB, 7/21 17:45:00, 30.71800, 5.47038E-003, 88.24927, 2.50000, 2.68984E-002, 0.00000
    - Zone Sizing Information, G SW APARTMENT, Heating, 1249.58808, 1499.50569, 6.63320E-002, 7.95984E-002, DENVER-AURORA-BUCKLEY.AFB_CO_USA ANN HTG 99.6% CONDNS DB, 12/21 09:30:00, -17.90000, 9.55238E-004, 88.24927, 2.50000, 2.68984E-002, 0.00000
    + Zone Sizing Information, G SW APARTMENT, Cooling, 1612.21726, 1934.66072, 0.14442, 0.17331, DENVER-AURORA-BUCKLEY.AFB_CO_USA ANN CLG .4% CONDNS DB=>MWB, 7/21 17:45:00, 30.71800, 5.47038E-003, 88.24927, 2.50000, 2.68984E-002, 0.00000
    + Zone Sizing Information, G SW APARTMENT, Heating, 1249.62041, 1499.54449, 6.63337E-002, 7.96005E-002, DENVER-AURORA-BUCKLEY.AFB_CO_USA ANN HTG 99.6% CONDNS DB, 12/21 09:30:00, -17.90000, 9.55238E-004, 88.24927, 2.50000, 2.68984E-002, 0.00000

**Comment by nrel-bot-2b** [comment]
@rraustad it has been 8 days since this pull request was last updated.

**Comment by nrel-bot-2** [comment]
@rraustad it has been 7 days since this pull request was last updated.

**Comment by nrel-bot-2b** [comment]
@rraustad it has been 12 days since this pull request was last updated.

File: src/EnergyPlus/HeatBalanceManager.cc (hunk 1 above)
Comment (Myoldmopar) [reply src/EnergyPlus/HeatBalanceManager.cc:4711]: I think this gets is close enough, personally.

**Review by Myoldmopar — APPROVED** [review]
Based on the discussion on the issue, this looks like the right fix.

**Comment by Myoldmopar** [comment]
There are some diffs here, but I would expect things to change very slightly with this.  Sizing values changing down in the decimal points is quite inline with expectations.  I don't think there's anything else to do here but merge.  I'll go ahead and do some extra local testing now, but will likely merge this shortly.

**Comment by Myoldmopar** [comment]
The only thing that changed in develop since I looked at this yesterday was the build warning fixes.  I'm good with this, merging.  Thanks @rraustad

**Pull request merged**